readthedocs_ext/readthedocs.py

```diff
@@ -126,6 +152,31 @@ def rtd_render(self, template, render_context):
                                                         app.builder.templates)
 
 
+def generate_json_artifacts(app, pagename, templatename, context, doctree):
+    try:
+        # We need to get the output directory where the docs are built
+        # _build/json
```

Comment:
Ok, I review a little more the code, this is copied to the artifacts folder in a later step, so we need to copy those there, or maybe do this in the `move_files` step, I go for the later